packages/epics/src/proposals/hooks/useProposals.ts

```diff
@@ -0,0 +1,72 @@
+import useSWR from 'swr';
+import { useState, useCallback } from 'react';
+import { data } from '../data.mock';
```

Comment:
I recommend to put your mocks as close to the location you're using them. I'd recommend `useProposal.mock.ts`